ocaml/xapi-consts/api_messages.ml

```diff
@@ -132,3 +132,7 @@ let host_cpu_features_down = addMessage "HOST_CPU_FEATURES_DOWN" 3L
 let host_cpu_features_up = addMessage "HOST_CPU_FEATURES_UP" 5L
 let pool_cpu_features_down = addMessage "POOL_CPU_FEATURES_DOWN" 5L
 let pool_cpu_features_up = addMessage "POOL_CPU_FEATURES_UP" 5L
+
+(* Cluster messages *)
+let cluster_host_creation_failed = addMessage "CLUSTER_HOST_CREATION_FAILED" 3L
+let cluster_host_enable_failed = addMessage "CLUSTER_HOST_ENABLE_FAILED" 4L
```

Comment:
If it failed the priority should not be 4, because 4 means it has recovered, see the table at the top of the file